athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/GetTableRequest.java

```diff
@@ -44,7 +47,7 @@ public class GetTableRequest
      * @param catalogName The catalog name that the table belongs to.
      * @param tableName The name of the table metadata is being requested for.
      */
-    @JsonCreator
+    @Deprecated
```

Comment:
so i don't think this can be deprecated directly since this is a top level Request object. If I remember correctly, this needs to always have 1 single constructor and we need to modify older request/response serDe's to take this into account.

Reply:
a good example is to take a look at the way we modified the ListTableRequest API where we added pagination token + page size.

Reply:
also, there should really be no reason to ever call the deprecated API anymore. I think we should delete.